lib/src/client.dart

```diff
@@ -10,13 +10,17 @@ class TwitchClient {
   TwitchClient({
     required this.clientId,
     required this.redirectUri,
+    this.clientSecret = "",
```

Comment:
I didn't put it in required because of compatibility and also when you create an application on Twitch developer you don't necessarily have a secret key you have to put the application in confidential (je suis fr si tu préfères)